packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -529,12 +529,11 @@ export class ARRenderer extends EventDispatcher {
       origin.sub(direction.multiplyScalar(model.idealCameraDistance));
       const ray = new Ray(origin, direction.normalize());
 
-      const modelToWorld = this.initialModelToWorld;
+      const modelToWorld = this.initialModelToWorld!;
```

Comment:
Does adding a `!` a the end clone the object or something?

Reply:
No, that's just telling TS don't bug me about this maybe being null, I know it's really there.

Reply:
Right, something about the new code seems fishy. Remind me the next time we sync.